lib/std/child_process.zig

```diff
@@ -151,10 +164,22 @@ pub const ChildProcess = struct {
         Unknown: u32,
     };
 
+    /// Behavior of the child process's standard input, output, and error
+    /// streams.
     pub const StdIo = enum {
+        /// Inherit the stream from the parent process.
         Inherit,
+
+        /// Ignore the stream.
```

Comment:
This just repeats the name of the field. Instead it should either be deleted or be replaced with something more descriptive such as "passes /dev/null as the stream"

Reply:
Updated.
I didn't change to active voice, though. The rest of the documentation in the file was not using it so I matched the style there.